src/OpenTelemetry/Metrics/MetricTypeExtensions.cs

```diff
@@ -22,12 +22,12 @@ public static class MetricTypeExtensions
     {
 #pragma warning disable SA1310 // field should not contain an underscore
 
-        internal const MetricType METRIC_TYPE_MASK = (MetricType)0xf0;
+        internal const MetricType METRIC_TYPE_MASK = (MetricType)0xff0;
 
         internal const MetricType METRIC_TYPE_SUM = (MetricType)0x10;
         internal const MetricType METRIC_TYPE_GAUGE = (MetricType)0x20;
-        internal const MetricType METRIC_TYPE_HISTOGRAM = (MetricType)0x30;
-        /* internal const byte METRIC_TYPE_SUMMARY = 0x40; // not used */
+        internal const MetricType METRIC_TYPE_HISTOGRAM = (MetricType)0x40;
+        /* internal const byte METRIC_TYPE_SUMMARY = 0x100; // not used */
```

Comment:
We can probably remove this (and the revert the 0xff0 part) since it's not used.